package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "nodeunit",
-  "version": "0.11.3",
+  "name": "ilib-nodeunit",
```

Comment:
Yes, that was the intention because nodeunit itself is deprecated so there will be no more new versions. In order to publish a new version from my userid, it would have to be under a different name.

BUT, this whole PR is moot now. The problem with the dependencies was resolved again, so now you can clone and npm install again as normal. We may need this repo in the future for ilib itself, but all of the other repos that don't need qt should get upgraded to a newer, more maintained testing library. It will be a giant task though.

Reply:
yes, I agree to consider changing another test framework if we don't need to support qt. I saw the message that the nodeunit is deprecated and they are strongly encouraged to use the other one.